storage/s3-sdk/list_gcs_objects.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+# [START storage_s3_sdk_list_objects]
+import boto3
+
+
+def list_gcs_objects(google_access_key_id, google_access_key_secret,
+                     bucket_name):
+    """Lists GCS objects using boto3 SDK"""
+    # Create a new client and do the following:
+    # 1. Change the endpoint URL to use the
+    #    Google Cloud Storage XML API endpoint.
+    # 2. Use Cloud Storage HMAC Credentials.
+
+    client = boto3.client("s3", region_name="auto",
+                          endpoint_url="https://storage.googleapis.com",
+                          aws_access_key_id=google_access_key_id,
+                          aws_secret_access_key=google_access_key_secret)
+
+    # Call GCS to list objects in bucket_name
+    response = client.list_objects(Bucket=bucket_name)
+
+    # Print object names
+    print("Objects:")
+    for bucket in response["Contents"]:
```

Comment:
Since this is listing objects in a bucket, it would be clearer for this to say `for object in ...` instead of `for bucket in ...`. (Or another word for object, just not bucket.)

Reply:
Or "blobs". That's a word I like.